libraries/provision/ansible/playbooks/stop-testserver-windows.yml

```diff
@@ -2,12 +2,12 @@
   hosts: windows
   vars:
     log_full_path:
-    custom_cbl_log_dir: C:\PROGRA~1\dotnet\TextLogging
 
   tasks:
-    - include: tasks/stop-fetch-windows-cbl-logs.yml
-
-    - name: Delete log file
-      win_file:
-        path: C:\PROGRA~1\dotnet\TextLogging
-        state: absent
+    - debug: msg="Stoping the Testserver "
+    - debug: msg="Copying logs at - {{ log_full_path }}"
+    - name: Stop TestServer
+      win_nssm:
+        name: TestServer
+        application: C:\PROGRA~1\dotnet\dotnet.exe
+        state: stopped
```

Comment:
why do you remove the log file clean up step? is the clean up step no longer needed?

Reply:
@euniceatcouchbase , we had a discussion and found that stop-testserver-windows is supposed to stop the Testserver and fetch logs, but When Manasa showed me the script, there is no stop for testServer , but deleting logs. I asked her to replace with stop TestServer and we do not want to remove the logs as in Teardown, we fetch the logs if there is failure. Also this stop-testserver can be used in the test stop and restart and we should not delete the logs during the test when we stop the testserver. 

AFter adding custom-logging feature, there are no logs getting generated under C:\PROGRA~1\dotnet\TextLogging. that is the reason failing, so it does not make send to try to remove log which does not exist

Reply:
got it, thanks for the explanation.